src/skills-builder/skills-builder-modal/view-results/ProductCardGrid.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CardGrid } from '@edx/paragon';
+import RecommendationCard from './RecommendationCard';
+
+const ProductCardGrid = ({
+  productTypeName, productTypeRecommendations, handleCourseCardClick, showAllButtonClickedList,
+}) => (
+  <CardGrid
+    columnSizes={{
+      xs: 12,
+      md: 6,
+      lg: 3,
+    }}
+    // TODO: figure out if this is working correctly
```

Comment:
Did you intend to leave this TODO?

Reply:
No I did not! Thanks for catching this. For context: This was in reference to the `hasEqualColumnHeights` attribute. I thought this attribute would force every card to have the same height. That is not what it does. If set to `false`, it will simply set `display: block;` for the wrapper of each Card in the grid.